### ISSUE TITLE
Missing styles cause failure to start

### ISSUE THREAD

**Issue by theJuddson** [ngbolt/ng-bolt#24]
```
[15:43:00] 'build' errored after 41 ms
[15:43:00] TypeError in plugin 'run-sequence(javascript)'
Message:
    Cannot read property 'sm' of undefined
Stack:
```

Occurs when the `breakpoints` attribute is omitted from styles.json: 
```
"breakpoints": {
  "sm": "0",
  "md": "600",
  "lg": "900",
  "xl": "1200",
  "xxl": "1800"
},
```

**Comment by theJuddson** [comment]
Reproduce: 

1. In ng-bolt source, remove the "breakpoint" attribute from the demo-app ./apps/demo-app/config/styles.json. 
2. Run `npm run demo-app`. 
3. Observe build error.

The offending gulp task in (gulp/javascript.js) needs to be updated to account for missing breakpoints. The task accounts for individual breakpoint entries that may be missing, but doesn't account for the entire attribute to be missing.

**Comment by theJuddson** [comment]
Fixed in v2.0.2
